src/SignalRServiceExtension/TriggerBindings/SignalRTriggerRouter.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    public class SignalRTriggerRouter
+    {
+        private readonly Dictionary<string, Dictionary<string, SignalRListener>> _listeners = new Dictionary<string, Dictionary<string, SignalRListener>>(StringComparer.OrdinalIgnoreCase);

```

Comment:
>_listeners [](start = 81, length = 10)

How about make it a class, so that we can easy know there's 2 levels dict, first hub level then method level.